### ISSUE TITLE
no way to build packages in aports/testing/ or with dependency on packages from same directory

### ISSUE THREAD

**Issue by OTLabs** [alpinelinux/docker-abuild#11]
The repository index for aports/testing/ is disabled by default so any intent to build a package with dependencies on package(s) from aports/testing/ will fail.

A test example on macOS:
```
cd aports/testing/zimwriterfs/
DABUILD_PACKAGES=~/packages/ ~/docker-abuild/dabuild -r
>>> zimwriterfs: Building testing/zimwriterfs 1.3-r0 (using abuild 3.4.0_rc3-r0) started Fri, 03 May 2019 02:07:57 +0000
>>> zimwriterfs: Checking sanity of /home/builder/aports/testing/zimwriterfs/APKBUILD...
>>> zimwriterfs: Analyzing dependencies...
ERROR: unsatisfiable constraints:
  gumbo-parser-dev (missing):
    required by: .makedepends-zimwriterfs-0[gumbo-parser-dev]
  libzim-dev (missing):
    required by: .makedepends-zimwriterfs-0[libzim-dev]
>>> ERROR: zimwriterfs: builddeps failed
>>> zimwriterfs: Uninstalling dependencies...
WARNING: Ignoring /home/builder/packages/main/x86_64/APKINDEX.tar.gz: No such file or directory
WARNING: Ignoring /home/builder/packages/community/x86_64/APKINDEX.tar.gz: No such file or directory
WARNING: Ignoring /home/builder/packages/testing/x86_64/APKINDEX.tar.gz: No such file or directory
```

**Comment by mor1** [comment]
Possibly addressed in 64685f096408eb56c5441fb2b593cdeedd35b63b -- let me know :)

**Comment by OTLabs** [comment]
Works like a charm! Thank you!
